src/cascadia/TerminalControl/dll/pch.cpp

```diff
@@ -0,0 +1,4 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"

```

Comment:
do we need one of these? can we just .. not have them at all?

Reply:
all the other libs have them just like this, totally empty. IIRC the project gets mad if it has _absolutely nothing_ to compile, so you gotta give it something (with nothing in it) to make it happy.